<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

margin: 0px;
padding: 0px;

}

#MainWindow{
background-color: rgb(239, 244, 240);

}


#Info_widget{
background-color: rgb(208, 232, 218);

}

#btnSave{
background-color: rgb(255, 255, 255);
color: rgb(39, 117, 75);
padding: 8px 30px

}



#tab{
border: 4px solid rgb(0, 0, 255)

}


#lblServerName{
border-bottom: 1px;
border-top-style: none;
border-bottom-style: solid;

}


#frame QPushButton{
background-color: rgba(255, 255, 255, 0);
padding: 8px 40px;

}


#frame QPushButton:hover{
border: 1px solid rgb(208, 232, 218);


}

#frame QPushButton:focus{
background-color:  rgb(208, 232, 218);
border: none;

}

#stackedWidget QPushButton{
background-color: rgb(255, 255, 255);
color: rgb(39, 117, 75);
border: 1px solid rgb(0, 0, 0)


}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnHome">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDBase">
         <property name="text">
          <string>Database</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDownload">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSetting">
         <property name="text">
          <string>Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pgHome">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Tab index 1 is on Database- button

Custom stylesheet is set in code. Else menu-button color is lost while push other buttons.

Todo: 
Fix all margins and layout -
*Have to strech all buttons to fit.
*Tab-widget CSS has to be fixed, so the tabs is not tabs, but blend into the page.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Home Panel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgDBase">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Install</string>
           </attribute>
           <widget class="QLabel" name="_lblOS">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>131</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>OS</string>
            </property>
           </widget>
           <widget class="QLabel" name="lblOS_info">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>80</y>
              <width>201</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Windows - 10 - 10.01.-177-SP0</string>
            </property>
           </widget>
           <widget class="QLabel" name="_lblQuickInstall">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>120</y>
              <width>211</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Quick install</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnQuickInstall">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>170</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Quick install</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnQuickUninstall">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>170</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Quick uninstall</string>
            </property>
           </widget>
           <widget class="QLabel" name="_lblStartOrStop">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>220</y>
              <width>201</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Start or Stop</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnStartServer">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>250</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Start Server</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnStopServer">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>250</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop Server</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnKillServer">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>250</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Kill Server</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="chkAutostart">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>300</y>
              <width>70</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Autostart</string>
            </property>
           </widget>
           <widget class="QLabel" name="_lblServerName">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>350</y>
              <width>201</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Server name:</string>
            </property>
           </widget>
           <widget class="QLabel" name="lblServerName">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>390</y>
              <width>201</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>MysqlServer56</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Basics</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Advanced</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Password</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgDownload">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Download Panel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgSetting">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Settings Panel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="Info_widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>MYSQL Status: Uninstalled</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnDBase</tabstop>
  <tabstop>btnDownload</tabstop>
  <tabstop>btnSetting</tabstop>
  <tabstop>btnHome</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnQuickInstall</tabstop>
  <tabstop>btnQuickUninstall</tabstop>
  <tabstop>btnStartServer</tabstop>
  <tabstop>btnStopServer</tabstop>
  <tabstop>btnKillServer</tabstop>
  <tabstop>chkAutostart</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
